pkg/chargeback/http_test.go

```diff
@@ -333,3 +335,282 @@ func TestAPIV1ReportsGet(t *testing.T) {
 		})
 	}
 }
+
+//for v2 endpoints
+func apiReportV2URL(reportName string) string {
+	return path.Join(APIV2Reports, reportName, "full")
+}
+
+func TestAPIV2ReportsGet(t *testing.T) {
+	const namespace = "default"
+	const testReportName = "test-report"
+	const testQueryName = "test-query"
+	reportStart := time.Time{}
+	reportEnd := reportStart.AddDate(0, 1, 0)
+
+	tests := map[string]struct {
+		reportStatus v1alpha1.ReportStatus
+		reportName   string
+
+		report      *v1alpha1.Report
+		query       *v1alpha1.ReportGenerationQuery
+		prestoTable *v1alpha1.PrestoTable
+
+		expectedStatusCode int
+		expectedAPIError   string
+
+		queryerPrepareFunc func(*mockpresto.MockExecQueryer) []presto.Row
+		expectedResults    *GetReportResults
+	}{
+		"report-finished-with-results-full": {
+			reportName: testReportName,
+			report:     newTestReport(testReportName, namespace, testQueryName, reportStart, reportEnd, v1alpha1.ReportStatus{Phase: v1alpha1.ReportPhaseFinished}),
+			query: newTestReportGenQuery(testQueryName, namespace, []v1alpha1.ReportGenerationQueryColumn{
```

Comment:
This query should have one field with tableHidden true, and one with false like the others to test that this is in fact returning columns with table hidden to true.

Reply:
Great catch! Thank you